app/src/test/java/waiter/ResponseBuilderTest.java

```diff
@@ -10,20 +10,26 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class ResponseFormatterTest {
+public class ResponseBuilderTest {
 
     @Property
     void responseProperlyFormatted(@ForAll @Size(4) List<@AlphaChars @NotBlank String> requestResults) {
-        Response Response = new Response(requestResults.get(0), requestResults.get(1), requestResults.get(2), requestResults.get(3));
+        Response response = new ResponseBuilder()
+                .newUp()
+                .protocol(requestResults.get(0))
```

Comment:
What do `requestResults` and `requestResults.get(0)` look like?

Reply:
They are just random strings acquired via jqwiks property-based-testing, just to show that it doesn't matter what is provided, it will be formatted correctly if that makes sense

Reply:
Interesting. I'm curious if status excepts a enum type, would property testing still apply here?

Reply:
Right. I think we might want to think about property tests as a supplement to the unit tests so they should be separated IMO. For this particular test, if we focus on the unit test aspect of it. It's not clear to me what those arguments are.

As for mocking enum, we don't want to mock `Enum` for the same reason we don't mock `Array` or `Map`.

Reply:
Yea I can pair for a bit today. My calendar should be updated.

Reply:
Make tests more clear and have them utilize the Enums, addressed in [this commit](https://github.com/jonmanahan/waiter/pull/18/commits/f82da021d65b0d3677ac9447063b627b01c5b6db)